$OpenBSD: patch-CMakeLists_txt,v 1.3 2015/10/01 08:43:12 jasper Exp $

- From 684b62e5407a5758b212a61ae5b624acc5325b7a Mon Sep 17 00:00:00 2001
  From: Robert Schroll <rschroll@gmail.com>
  Date: Wed, 3 Jun 2015 16:20:28 -0400
  Subject: [PATCH] Remove Vala_CMake submodule; include cmake module directly

- From cea8603540e7994bcf72a92a4faf6ffc7fff7c62 Mon Sep 17 00:00:00 2001
  From: Jasper Lievisse Adriaanse <jasper@humppa.nl>
  Date: Wed, 30 Sep 2015 19:37:09 +0200
  Subject: [PATCH] Allow for specifying the manpage installation dir

--- CMakeLists.txt.orig	Mon Jun  1 04:37:51 2015
+++ CMakeLists.txt	Wed Sep 30 19:34:12 2015
@@ -1,12 +1,13 @@
 project("pdfpc" C)
 cmake_minimum_required(VERSION 2.6)
 
-list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/Vala_CMake/vala)
+list(APPEND CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake/vala)
 
 find_package(Vala "0.26" REQUIRED)
 include(${VALA_USE_FILE})
 
 set(SYSCONFDIR "${CMAKE_INSTALL_PREFIX}/etc" CACHE FILEPATH "sysconfdir")
+set(MANDIR "share/man/man1" CACHE FILEPATH "mandir")
 
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
